### DISCUSSION TITLE
Cycleway overlay: cycle track with walking allowed

### DISCUSSION THREAD

**Discussion by olecky** [streetcomplete/StreetComplete#5990] [Feedback]
When editing in the cycleways overlay, there doesn't seem to be an option to select cycle track (signed with [D4](https://commons.wikimedia.org/wiki/File:Vienna_Convention_road_sign_D4a.svg) or similar sign), respecting the fact that pedestrians are allowed to use it because of national or international legislation. In most cases, if there is no sidewalk, pedestrians are allowed to walk on the cycle track (article 20 paragraph 3 of the Convention on Road Traffic). Example:

![IMG_20200927_145527-resized](https://github.com/user-attachments/assets/2ae7e379-837b-4f15-b57c-4711ae1ccb50)

In the app, the closest I can choose are:

- EXCLUSIVE - which adds foot=no, banning pedestrians
- NON_SEGREGATED - which adds foot=designated, implying that there is sign designating the highway for pedestrians (so a combination of D4 and D5 rather than only D4)

Both of the option seem incorrect to me in the example shown on the picture. I'm not sure if it's necessary to add one more option to distinguish cycle tracks with walking allowed and not. From my point of view, removing the foot=no addition in EXCLUSIVE would solve the problem. Is it really useful for pedestrian routing in situations where there is parallel cycle track and footpath?

**Comment by westnordost** [comment]
You are right, I think your suggestion makes sense.

**Comment by westnordost** [comment]
Posted as issue.

**Reply by matkoniecz** [reply]
https://github.com/streetcomplete/StreetComplete/issues/5991
